docs/api_docs/bundle.yaml

```diff
@@ -863,6 +863,9 @@ definitions:
       key:
         description: unique key representation of the flag
         type: string
+      template:
```

Comment:
this bundle.yaml is generated, I think can change this file and then `make gen` https://github.com/checkr/flagr/blob/848bcf07b8ff4f6a4a056794d521c62535b2e3b8/swagger/index.yaml#L140-L145